.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -41,6 +41,14 @@ body:
     validations:
       required: true
 
+  - type: textarea
```

Comment:
This sort of duplicates the "Current behavior" block. However, it is true that nothing specifically talks about reproduction, which is not good.

However, I'd rather see your text combined with the description of `behavior-current` rather than making the annoying form even more annoying.

Reply:
i actually had the same thought, but then one can make an even stronger case for collapsing the "actual" and "expected" blocks.

but if we approach this from the "guidance" perspective, then it actually makes sense to atomize it, to force the submitter to think about what is cause and what is effect, and possibly consider implicit assumptions.

Reply:
> but if we approach this from the "guidance" perspective, then it actually makes sense to atomize it, to force the submitter to think about what is cause and what is effect, and possibly consider implicit assumptions.

Ditto, here is the latest (positive) example: https://github.com/MidnightCommander/mc/issues/4670